creature_factory.cpp

```diff
@@ -1068,13 +1068,19 @@ CreatureAttributes CreatureFactory::getAttributesFromId(CreatureId id) {
     case CreatureId::CYCLOPS: 
       return CATTR(
           c.viewId = ViewId::CYCLOPS;
-          c.attr = LIST(34_dam, 40_def );
-          c.body = Body::humanoid(Body::Size::LARGE);
+          //He is just too big to wear armour and hold little weapons
+          //In all honesty, capturing him was fun but with full equipment
+          //he was very over powered.
+          c.cantEquip = true; 
+          c.attr = LIST(40_dam, 40_def );
+          c.body = Body::humanoid(Body::Size::HUGE);
           c.body->setWeight(400);
           c.permanentEffects[LastingEffect::RANGED_RESISTANCE] = 1;
+          c.permanentEffects[LastingEffect::HALLU] = 1;
+          c.permanentEffects[LastingEffect::INSANITY] = 1;
```

Comment:
I like this pull request. How does the insanity effect work when you capture him though, won't he kill all your minions?